### DISCUSSION TITLE
On the harvester, how do you know if the full node is connected

### DISCUSSION THREAD

**Discussion by hyzgit** [Chia-Network/chia-blockchain#7913] [Q&A]
On the harvester, how to know whether the harvester is working properly, such as whether the full node has been connected, the number of plots files, etc.

**Comment by keliew** [comment]
https://github.com/Chia-Network/chia-blockchain/wiki/How-to-Check-If-Everything-is-Working-(or-Not)#harvester
Good starting point.

**Reply by hyzgit** [reply]
I found a working harvester and executed the following command. There was no mission information output. I think this is not a good idea.
Select-String -Path "~\.chia\mainnet\log\debug*" -Pattern "eligible"
Select-String -Path "~\.chia\mainnet\log\debug*" -Pattern "Found [^0] proof"
Select-String -Path "~\.chia\mainnet\log\debug*" -Pattern "Farmed unfinished_block"
Get-Content -Path "~\.chia\mainnet\log\debug.log" -Wait | Select-String -Pattern "found"

Is there a more direct way, like through the CLI，For example, if we want to know the state of the farm, we will run "chia farm summary" instead of looking for the logs file. Does harvester have a similar command?

**Reply by rmica** [reply]
It doesn't. A working harvester is receiving challenges from the farmer (and writing log infos) at a ~10 seconds pace, and has a total plots count consistent with your local drives. To monitor an harvester is a way to monitor whole farming process health. In my case I collect timeseries and challenges have to stay around 30 in a 5 minutes range. I know, it's not easy for high end users, but it's what I do, if it'll help.

**Reply by rmica** [reply]
You could also evaluate third party tools like chiadog or chiamonitor. Always look twice at what you install and run. The tools I mentioned are open and audited logfile followers.
